flags/14.0/vtgate.txt

```diff
@@ -1,48 +1,47 @@
       --buffer_max_failover_duration duration                            Stop buffering completely if a failover takes longer than this duration. (default 20s)
       --buffer_min_time_between_failovers duration                       Minimum time between the end of a failover and the start of the next one (tracked per shard). Faster consecutive failovers will not trigger buffering. (default 1m0s)
       --buffer_size int                                                  Maximum number of buffered requests in flight (across all ongoing failovers). (default 1000)
       --buffer_window duration                                           Duration for how long a request should be buffered at most. (default 10s)
       --catch-sigpipe                                                    catch and ignore SIGPIPE on stdout and stderr if specified
-      --cell string                                                      cell to use (default test_nj)
+      --cell string                                                      cell to use (default "test_nj")
       --cells_to_watch string                                            comma-separated list of cells for watching tablets
       --consul_auth_static_file string                                   JSON File to read the topos/tokens from.
-      --cpu_profile string                                               deprecated: use '-pprof=cpu' instead
       --datadog-agent-host string                                        host to send spans to. if empty, no tracing will be done
       --datadog-agent-port string                                        port to send spans to. if empty, no tracing will be done
-      --dbddl_plugin string                                              controls how to handle CREATE/DROP DATABASE. use it if you are using your own database provisioning service (default fail)
-      --ddl_strategy string                                              Set default strategy for DDL statements. Override with @@ddl_strategy session variable (default direct)
-      --default_tablet_type value                                        The default tablet type to set for queries, when one is not explicitly selected (default PRIMARY)
-      --disable_local_gateway                                            deprecated: if specified, this process will not route any queries to local tablets in the local cell
-      --discovery_high_replication_lag_minimum_serving duration          the replication lag that is considered too high when applying the min_number_serving_vttablets threshold (default 2h0m0s)
-      --discovery_low_replication_lag duration                           the replication lag that is considered low enough to be healthy (default 30s)
-      --emit_stats                                                       If set, emit stats to push-based monitoring and stats backends
```

Comment:
This looks very wrong to me. We have kept the `statsd_address` etc flags, which eventually use this flag value to know whether to emit stats on a regular interval or not.

Reply:
Fixed in #11450

Reply:
This is fixed as part of the stats package fix.